toolchain/check/import_cpp.cpp

```diff
@@ -427,6 +444,28 @@ static auto GetReturnType(Context& context, SemIR::LocId loc_id,
   return param_pattern_id;
 }
 
+// Returns parameter patterns block id, the return slot pattern id, and the
+// call parameters block id for the given function declaration.
+// Returns false if the function declaration has an unsupported parameter type.
+static auto GetFunctionParams(Context& context, SemIR::LocId loc_id,
+                              clang::FunctionDecl* clang_decl,
+                              SemIR::InstBlockId& param_patterns_id,
+                              SemIR::InstId& return_slot_pattern_id,
+                              SemIR::InstBlockId& call_params_id) -> bool {
```

Comment:
Consider returning all of these instead of using output parameters.
You can have a dedicated struct for that.
You can make it optional to clarify success or add a bool to the struct if you want to be able to use some of the values even on failure.

From [the style guide](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs):
> Prefer using return values over output parameters